agagd/templates/agagd_core/chapter.html

```diff
@@ -46,7 +46,7 @@ <h2> Chapter: {{ chapter.code }} </h2>
       <dt> Website: </dt> <dd> <a href="{{chapter.url}}">{{ chapter.url }}</a> </dd>
   {% endif %}
 </dl>
-
+<br><br>
```

Comment:
I don't mind this, but I think we could do with one here.

New
![image](https://user-images.githubusercontent.com/5054653/103594468-a1746280-4ead-11eb-93b3-4993279b8a95.png)

Old
![image](https://user-images.githubusercontent.com/5054653/103594487-ac2ef780-4ead-11eb-85ba-498824c64c4e.png)

Reply:
One Two, As long as theres a slight break between the two I think it looks better to break up the sections. Unlike the odd way that last sentence is structured.

Reply:
I have updated my comment: https://github.com/usgo/agagd/pull/140#discussion_r551656508 to make more sense.